ex03.ex

```diff
@@ -55,7 +55,19 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+def odd_even([]), do: []
```

Comment:
# -4

This only works for numbers between 1 and 9.

I was looking for something like:

~~~ elixir
  def odd_even([]), do: []
  def odd_even([ h | t ]), do: [ atom_for(h) | odd_even(t) ]

  defp atom_for(n) when Integer.is_odd(n),  do:  :odd
  defp atom_for(n) when Integer.is_even(n), do:  :even
~~~